app/grandchallenge/algorithms/tasks.py

```diff
@@ -478,3 +484,80 @@ def set_credits_per_job():
             algorithm.credits_per_job = default_credits_per_job
 
         algorithm.save(update_fields=("credits_per_job",))
+
+
+@shared_task(**settings.CELERY_TASK_DECORATOR_KWARGS["acks-late-micro-short"])
```

Comment:
More likely this needs to go on the large queue as I'm not confident it will always fit into 5 minutes.